.travis.yml

```diff
@@ -3,10 +3,11 @@ language: node_js
 node_js:
   - "0.10"
   - "4"
+  - "5"
 
 matrix:
   allow_failures:
-    - node_js: "4"
```

Comment:
@benbp if it don't break anything, it make sense that support nodejs 4 more strongly.